<launch>
    <!-- <node pkg="semantic_knowledgebase" type="wall_publisher_from_file.py" name="wall_publisher" output="screen">
         <param name="filename" type="string" value="$(find semantic_knowledgebase)/data/wall_list.yaml" />
    </node> -->

    <node pkg="semantic_knowledgebase" type="object_publisher.py" name="object_information" output="screen">
         <param name="filename" type="string" value="$(find semantic_knowledgebase)/data/object_list.yaml" />
    </node> 

</launch>
